app/src/screens/search-flow/QuestionSummary.tsx

```diff
@@ -35,6 +35,17 @@ function QuestionSummary({item, index, data, generate_ai_summary}: Props) {
   const [stats, setStats] = useState<Statistics | null>(null);
   const [modalActive, setModalActive] = useState<boolean>(false);
 
+  const processedData = useMemo(
```

Comment:
I think a system where you average duplicate data points would be safer for the data, since we don't know which individual report was the most accurate.